### PR TITLE
pls im dyng

### PR DESCRIPTION
**Pull request by RatFromTheJungle**
## About The Pull Request

i dont want die (before anyone screeches, look where it's directed towards)

## Why It's Good For The Game
every assistant has loud stompy noises, and it gets a little crazy when there's like 4 of them in a room running in circles

### CODE REVIEW

File: code/modules/jobs/job_types/head_of_security.dm [hunk 1]
```diff
@@ -50,7 +50,7 @@
 	belt = /obj/item/pda/heads/hos
 	ears = /obj/item/radio/headset/heads/hos/alt
 	uniform = /obj/item/clothing/under/rank/security/head_of_security
-	shoes = /obj/item/clothing/shoes/jackboots
+	shoes = /obj/item/clothing/shoes/jackboots/sec
```
Comment (FlamingLily) [inline code/modules/jobs/job_types/head_of_security.dm]:
```suggestion
	shoes = /obj/item/clothing/shoes/jackboots/sec // SKYRAT EDIT CHANGE
```

File: code/modules/jobs/job_types/security_officer.dm [hunk 2]
```diff
@@ -205,7 +205,7 @@ GLOBAL_LIST_EMPTY(security_officer_distribution)
 	gloves = /obj/item/clothing/gloves/color/black
 	head = /obj/item/clothing/head/helmet/sec
 	suit = /obj/item/clothing/suit/armor/vest/alt
-	shoes = /obj/item/clothing/shoes/jackboots
+	shoes = /obj/item/clothing/shoes/jackboots/sec
```
Comment (FlamingLily) [inline code/modules/jobs/job_types/security_officer.dm]:
```suggestion
	shoes = /obj/item/clothing/shoes/jackboots/sec // SKYRAT EDIT CHANGE
```

File: code/modules/jobs/job_types/warden.dm [hunk 3]
```diff
@@ -48,7 +48,7 @@
 	belt = /obj/item/pda/warden
 	ears = /obj/item/radio/headset/headset_sec/alt
 	uniform = /obj/item/clothing/under/rank/security/warden
-	shoes = /obj/item/clothing/shoes/jackboots
+	shoes = /obj/item/clothing/shoes/jackboots/sec
```
Comment (FlamingLily) [inline code/modules/jobs/job_types/warden.dm]:
```suggestion
	shoes = /obj/item/clothing/shoes/jackboots/sec // SKYRAT EDIT CHANGE
```

**Comment by FlamingLily** [comment]
Nt rep also gets jackboots, and so did sarge iirc, be sure to update them accordingly

**Comment by Gandalf2k15** [comment]
OR just make loadout jackboots a civilian subtype.

**Comment by Gandalf2k15** [comment]
This isn't even a thing anymore

**Comment by GoldenAlpharex** [comment]
> This isn't even a thing anymore

I mean, on the server right now, we can easily hear all jackboots creaking.

**Comment by RatFromTheJungle** [comment]
it is actually not a thing anymore

**Pull request closed without merging**